### ISSUE TITLE
Add favicon for `qute://` pages

### ISSUE THREAD

**Issue by saro** [qutebrowser/qutebrowser#3755]
Personally I rely on favicon to identify the tabs in the vertical bar.
It would be very nice to have at least one icon to identify all the `qute://` pages.

**Comment by juliandorville** [comment]
Hi, I'd love to work on this issue!

**Comment by jgkamat** [comment]
Hi @juliandorville! I would like to see this done too, but I personally don't know where to start with this (I'm not sure if it's possible on qtwebengine).

I took a quick look at [qutescheme](https://github.com/qutebrowser/qutebrowser/blob/master/qutebrowser/browser/qutescheme.py) and [webenginequtescheme](https://github.com/qutebrowser/qutebrowser/blob/master/qutebrowser/browser/webengine/webenginequtescheme.py), but I wasn't able to find any place (or api) where we can specify a favicon. From the api, it looks like we provide some html and that shows up in the page when we hit that URL. Because it's a bit unclear, this might not be a good place for a first contribution, unless I'm missing something myself...

We could do something hacky like detecting if we are in `qute` pages and forcibly overriding the favicon to the qb logo, but that seems unideal to me. Maybe @The-Compiler has a better idea of how to tackle this.

**Comment by toofar** [comment]
I could be completely off base but I would suggest adding an icon handler in `qutebrowser/browser/qutescheme.py` to return one of the things in `icons/` and then adding a link rel tag to `qutebrowser/html/base.html`. You would have to check at least

* where the icons are installed to, if at all, so the qutescheme handler could find them
* that all the qutescheme pages eventually inherited from `base.html`

**Comment by juliandorville** [comment]
Yes, @toofar , that’s pretty much what I had in mind, too. But maybe we’re missing something?

**Comment by JimenezJC** [comment]
Hello, I've been working on this issue and seem to have run into a block. I've added an image handler to call the icon, and calling the icon using `qute://` loads the image in the browser, and the image seems to be called whenever i use `img src` however, when calling it using the `<link rel = "icon' .. >` tag the image does NOT load.

I'm not entirely sure why that is since my handler manages to call the icon just fine, interestingly enough when i use `<link rel="shortcut icon" type ="image/png" href="https://qutebrowser.org/media/favicon.png" />` the favicon does work. This is a possible solution but i'm not sure if is the best one?
